xen/drivers/passthrough/iommu.c

```diff
@@ -142,7 +142,14 @@ int iommu_domain_init(struct domain *d, bool_t use_iommu)
         return 0;
 
     hd->platform_ops = iommu_get_ops();
-    return hd->platform_ops->init(d);
+    ret = hd->platform_ops->init(d);
```

Comment:
please reword [commit](https://github.com/xen-troops/xen/pull/48/commits/8bf040bd5d28fc2d7417609187f9ad79e4b601a1) message to be more detailed